apps/docs/src/api/sandbox/route.ts

```
@@ -0,0 +1,161 @@
+import { APIClient, sandboxRun, getServiceUrls } from '@agentuity/server';
+import { Writable } from 'node:stream';
+import { SCRIPTS, SCRIPT_DEFAULTS } from './scripts';
+
+const router = createRouter();
+
+const SNAPSHOT_ID = process.env.SANDBOX_SNAPSHOT_ID;
+const SANDBOX_EXEC_TIMEOUT = '2m';
+const AI_GATEWAY_URL = 'https://catalyst.agentuity.cloud/gateway';
+
+function cleanOutput(content: string): string {
+	let cleaned = content.replace(/\x1b\[[0-9;]*m/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)